api/src/main/java/org/openmrs/validator/DrugOrderValidator.java

```diff
@@ -137,17 +136,7 @@ else if(order.getDrug() != null && order.isNonCodedDrug()){
 		}
 	}
 	
-	private void validateFieldsForOutpatientCareSettingType(DrugOrder order, Errors errors) {
```

Comment:
I'm a bit confused - can you explain why you are removing this validation?

Reply:
> I'm a bit confused - can you explain why you are removing this validation?

well the first error we get is due to this validation

Reply:
Right.  But does this mean we never check this aspect of the validation?  This validation rule is still valid, it just needs to be enforced after all of the different order properties have been set.